x-pack/plugins/ml/public/application/util/index_utils.ts

```diff
@@ -129,6 +129,6 @@ export function timeBasedIndexCheck(dataView: DataView, showNotification = false
  * Returns true if the data view index pattern contains a :
  * which means it is cross-cluster
  */
-export function isCcsIndexPattern(dataViewIndexPattern: string) {
-  return dataViewIndexPattern.includes(':');
+export function isCcsEsIndexPattern(esIndexPattern: string) {
```

Comment:
I don't think it's necessary have the prefix `es` in these variable names. 
With this change, all references to `indexPattern` will be obvious that it’s referring to the sting which matches indices.
In other places in the kibana codebase they don't use a prefix e.g. https://github.com/elastic/kibana/blob/426a379f311fe76fccfe1fbf260ca0ea1b41f772/src/plugins/data_views/common/data_views/abstract_data_views.ts#L207C8-L207C8

Reply:
Agree to go just with `indexPattern` for consistency, I reverted the changes in [fe60e8d](https://github.com/elastic/kibana/pull/171820/commits/fe60e8df718f8722df02edb09f9c52d73b518fdc) and [958d29c](https://github.com/elastic/kibana/pull/171820/commits/958d29c778997288b69ca73ce92c522cf4b0e533).